### DISCUSSION TITLE
Peers which go offline too often render the channels inactive for routing.

### DISCUSSION THREAD

**Discussion by von-Hodl** [Ride-The-Lightning/RTL#799] [Q&A] [answered]
I'm curious about this.

"Peers which go offline too often render the channels inactive for routing."

What is the process that renders them inactive from routing? What qualifies you as being offline too much?

And if your inactive for routing can you do something to make it better?

**Comment by saubyk** [comment, marked as answer]
- When your peer which you have an open channel with goes offline, the channel goes into an inactive state. Which means that the channel is not usable for any payment or routing activity.
- Peers which goes offline too often are not good peers for routing, because when the channel is inactive, your channel liquidity is also inactive, which could potentially have been utilized for routing elsewhere.
- Ideally, you need some sort of a tracking solution to measure the time period for which a node is offline. And then decide what thresholds you want to tolerate for your peers to be offline. Based on those thresholds you can decide to close channels with the peers.
- The options you have for dealing with offline peers are limited:
  - if you know the peer, you can reach out and ask them to bring the node up again
  - you can force close the channels and take your liquidity elsewhere. But force closing has a cost and you may have to wait a bit until the funds show up on your node.

**Reply by von-Hodl** [reply]
Ok so there isn't an automated process in Lightning that will "mark" a peer as "not good for routing"

It's an manual process of looking at your peers and your peers looking at you and making a choice to close a channel.

I'm just clarifying. To isolate potential causes for the current lack of routing I'm having. Making sure my node wasn't identified by some automated process to be excluded from routed.

**Reply by saubyk** [reply]
The definition of what's 'not good for routing' can be subjective, should take a couple of factors into account, not just node availability. Although scoring mechanisms are being developed to make it more measurable and objective.
There is a node scoring method which Lightning Labs has developed (called BOS score) but it's not open source and opaque on how its calculated.

BOS score is used to determine the eligibility of participation on the pool product.

I am not sure if LND is taking BOS score into account when its calculating routes for payment routing, but if it does and your BOS score is low, it can impact the probability of payments getting forwarded through your node.
You can read more about BOS score here:
https://lightningwiki.net/index.php/BOS_Score

**Reply by von-Hodl** [reply]
Interesting. My node doesn't appear to have a BOSS score.
